02LessonTwo/homework/index.html

```diff
@@ -11,12 +11,15 @@
 
 <body bgcolor="#6ecff6">
 
-<h1 style="color:blue">Daniel Cisneros</h1>
+
+<h1 style="color: blue">Daniel Cisneros</h1>
 <hr> <!--Add hr is an important step-->
   <p style:"font-family: Times, serif; font-size: 12px;">1413 Dirt Lane, Austin Tx.</p>
 
```

Comment:
be careful when indenting. the only times we indent is to indicate a parent>child relationship. in this case and in several more in the page, you have a paragraph indented like a child of an h2 or hr when they are only children of the div.

Also, try adding in a section or main!